sdv/_utils.py

```diff
@@ -301,63 +307,81 @@ def _check_is_lower_version(current_version, synthesizer_version):
         try:
             current_v = int(current_v)
             synth_v = int(synth_v)
-            if current_v > synth_v:
+            if compare_operator(current_v, synth_v):
                 return False
 
-            if synth_v > current_v:
+            if compare_operator(synth_v, current_v):
```

Comment:
what happens when the versions have .dev0 or r1 in them?

Reply:
We do not compare that. If we have `dev0` vs`dev1` for example, it will fail and return `False` (because of the `try except`), I thought that we wanted something basic, not going that depth. If thats the case the best approach is to just use `packaging` which compares everything (dev0 to dev1, dev0 to r1 even rc1 and other nomenclatures).